psalm.xml

```diff
@@ -14,4 +14,11 @@
             <directory name="vendor" />
         </ignoreFiles>
     </projectFiles>
+        <issueHandlers>
+        <PropertyNotSetInConstructor>
```

Comment:
Can we add comment like in this PR [mot-google-analytics](https://github.com/dvsa/mot-google-analytics/pull/3/files#diff-09c70a066977bc3c14f388e00c07d2b98e54f44311c61651a055ec89b5798623R18):
```suggestion
<!-- Suppress the PropertyNotSetInConstructor error in the test folder only. -->
        <!-- Many of the test files extend AbstractControllerTestCase which uses a few unset properties -->
        <!-- that are not set in a constructor, but are legitimate. -->
        <PropertyNotSetInConstructor>
```

Reply:
Certainly, I've C&P'd that comment into the file